Update the documentation for max_override_period_minutes (#511)
seamapi/docs: docs/capability-guides/thermostats/creating-and-managing-thermostat-schedules.md

Each thermostat schedule can contain the following properties:

> <table><thead><tr><th width="339">Property</th><th>Description</th></tr></thead><tbody><tr><td><code>name</code></td><td>(Optional) User-friendly name to identify the thermostat schedule.</td></tr><tr><td><code>climate_preset_key</code></td><td>(Required) Key of the climate preset to use for the thermostat schedule.</td></tr><tr><td><code>max_override_period_minutes</code></td><td>Number of minutes for which a person at the thermostat can change the thermostat's settings after the activation of the scheduled climate preset.<br><strong>Note:</strong> Setting this value to <code>0</code> means that the override will apply until another climate preset becomes active (i.e. the override timeout is disabled).<br>Default: <code>0</code><br>See also <a href="creating-and-managing-thermostat-schedules.md#specifying-manual-override-permissions">Specifying Manual Override Permissions</a>.</td></tr><tr><td><code>starts_at</code></td><td>(Required) Date and time at which the thermostat schedule starts, in <a href="https://www.iso.org/iso-8601-date-and-time-format.html">ISO 8601</a> format.</td></tr><tr><td><code>ends_at</code></td><td>(Required) Date and time at which the thermostat schedule ends, in <a href="https://www.iso.org/iso-8601-date-and-time-format.html">ISO 8601</a> format.</td></tr></tbody></table>

Reviewer: ```suggestion
<table><thead><tr><th width="339">Property</th><th>Description</th></tr></thead><tbody><tr><td><code>name</code></td><td>(Optional) User-friendly name to identify the thermostat schedule.</td></tr><tr><td><code>climate_preset_key</code></td><td>(Required) Key of the climate preset to use for the thermostat schedule.</td></tr><tr><td><code>max_override_period_minutes</code></td><td>Number of minutes for which a person at the thermostat or using the API can change the thermostat's settings after the activation of the scheduled climate preset.<br><strong>Note:</strong> If you set this value to <code>0</code>, the override applies until another climate preset becomes active, that is, the override timeout is disabled.<br>Default: <code>0</code><br>See also <a href="creating-and-managing-thermostat-schedules.md#specifying-manual-override-permissions">Specifying Manual Override Permissions</a>.</td></tr><tr><td><code>starts_at</code></td><td>(Required) Date and time at which the thermostat schedule starts, in <a href="https://www.iso.org/iso-8601-date-and-time-format.html">ISO 8601</a> format.</td></tr><tr><td><code>ends_at</code></td><td>(Required) Date and time at which the thermostat schedule ends, in <a href="https://www.iso.org/iso-8601-date-and-time-format.html">ISO 8601</a> format.</td></tr></tbody></table>
```
